src/modules/check.js

```diff
@@ -0,0 +1,33 @@
+const check = (obj) => {
+  const checkbox = document.getElementById(`checkbox-${obj.index}`);
```

Comment:
Great work separating your js files based on functionality, but I would suggest that you avoid defining the same constant more than once, instead, you can define it globally and use it in the function scope. that will make your code look cleaner

Reply:
Thank you! @shakerAbuDrais I'll work on that